app/models/jobsModel.js

```diff
@@ -16,20 +16,21 @@ async function insert(jobs){
         await db.transaction(async function(trx) {
             try {
                 const clients = job.clients;
-                const job_entry = {
+                const jobEntry = {
                     class_id : job.class_id,
+                    class_schedule_id : classJson.ClassScheduleId,
```

Comment:
I might've missed it but is the classJson variable something you don't have yet?

Reply:
Good catch, thanks!